addons/sale_mrp/tests/test_multistep_manufacturing.py

```diff
@@ -10,21 +10,21 @@ class TestMultistepManufacturing(TestMrpCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-
-        # Required for `uom_id ` to be visible in the view
-        cls.env.user.groups_id += cls.env.ref('uom.group_uom')
-        # Required for `manufacture_steps` to be visible in the view
-        cls.env.user.groups_id += cls.env.ref('stock.group_adv_location')
-        # Required for `product_id` to be visible in the view
-        cls.env.user.groups_id += cls.env.ref('product.group_product_variant')
+        with cls.sudo():
+            cls.user.groups_id += (
+                # Required for `manufacture_steps` to be visible in the view
+                cls.env.ref('stock.group_adv_location') +
+                # Required for `product_id` to be visible in the view
+                cls.env.ref('product.group_product_variant') +
+                # TODO: these are required for some tests, is this valid?
+                cls.env.ref('account.group_account_readonly') +
+                cls.env.ref('analytic.group_analytic_accounting') +
```

Comment:
Hmm. It's better to check:
- It happens once: maybe it should only be added in the test
- Maybe it's an issue in the code
- Maybe the user need the right for the feature